### PR TITLE
Add a11y-features to dropdown-components

### PR DESCRIPTION
**Pull request by aurorawalberg**
# Describe your changes

- Make the dropdown components MdSelect, MdMultiSelect, and MdAutocomplete accessible: close on escape press and re-focus initial element, tab through content closes dropdown
- Add focus states for MdMulstiSelect and MdAutocomplete
- Fix MdMultiSelect enter press not selecting the focused element

## Checklist before requesting a review

- [x] I have performed a self-review and test of my code
- [ ] If new component: Is story for component created in `stories`-folder?
- [ ] If new component: Is tsx-file import added to `packages/react/index.tsx`?
- [ ] If new component: Is css-file added to `packages/css/index.css`?

### CODE REVIEW

File: packages/react/src/hooks/useDropdown.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    if (e.key === 'Tab') {
+      if (e.shiftKey) {
+        return;
+      }
+      if (!focusableElements || !document.activeElement) {
+        return;
+      }
+      const lastElement = focusableElements[focusableElements.length - 1];
```
Comment (kajsaeggum) [inline packages/react/src/hooks/useDropdown.ts:27]: Mulig her at focusableElements.length - 1 kan bli -1 hvis arrayen er tom?

File: packages/react/src/formElements/MdMultiSelect.tsx [hunk 2]
```diff
@@ -167,13 +171,16 @@ const MdMultiSelect: React.FunctionComponent<MdMultiSelectProps> = ({
...
 
       <MdClickOutsideWrapper
+        ref={dropdownRef}
         onClickOutside={() => {
           return setOpen(false);
         }}
         className="md-multiselect__dropdown-wrapper"
       >
         <button
+          role="listbox"
```
Comment (kajsaeggum) [inline packages/react/src/formElements/MdMultiSelect.tsx:181]: Denne bør ha role="combobox", og så er det listen med options som skal ha rollen listbox.

Comment (aurorawalberg) [reply packages/react/src/formElements/MdMultiSelect.tsx:181]: Aha! Tusen takk.

File: packages/react/src/hooks/useDropdown.ts (hunk 1 above)
Comment (aurorawalberg) [reply packages/react/src/hooks/useDropdown.ts:27]: Ja, nettopp. Takk.

File: packages/react/src/formElements/MdSelect.tsx [hunk 3]
```diff
@@ -154,16 +157,22 @@ const MdSelect = React.forwardRef<HTMLButtonElement, MdSelectProps>(
...
           className="md-select__container"
         >
           <button
+            aria-labelledby={`md-select_label_${uuid}`}
+            role="listbox"
             id={`md-select_${uuid}`}
             aria-describedby={helpText && helpText !== '' ? `md-select_help-text_${uuid}` : undefined}
             className={buttonClassNames}
             type="button"
+            aria-activedescendant={
```
Comment (kajsaeggum) [inline packages/react/src/formElements/MdSelect.tsx:173]: aria-activedescendant skal ikke peke på option som er selected, men option som akkurat nå har fokus. Altså at den endres mens man blar mellom options (uten å nødvendigvis velge noen av de)

Comment (aurorawalberg) [reply packages/react/src/formElements/MdSelect.tsx:175]: Aha! Fikser.

**Review by kajsaeggum — APPROVED** [review]

**Pull request merged**